services/ui-src/src/forms/mcpar/mcpar.json

```
@@ -245,63 +245,63 @@
             "id": "bedr",
             "fields": [
               {
-                "id": "bedr-1",
+                "id": "encounterDataValidationEntity",
                 "type": "checkbox",
                 "validation": "checkbox",
                 "props": {
                   "label": "B.III.1 Data validation entity",
                   "hint": "Select the state agency/division or contractor tasked with evaluating the validity of encounter data submitted by MCPs.</br>Encounter data validation includes verifying the accuracy, completeness, timeliness, and/or consistency of encounter data records submitted to the state by Medicaid managed care plans. Validation steps may include pre-acceptance edits and post-acceptance analyses.",
                   "choices": [
                     {
-                      "name": "bedr-1-o1",
+                      "name": "encounterDataValidationEntity-o1",
                       "type": "choice",
                       "label": "State Medicaid agency staff"
                     },
                     {
-                      "name": "bedr-1-o2",
+                      "name": "encounterDataValidationEntity-o2",
                       "type": "choice",
                       "label": "Other state agency staff"
                     },
                     {
-                      "name": "bedr-1-o3",
+                      "name": "encounterDataValidationEntity-o3",
                       "type": "choice",
                       "label": "State actuaries"
                     },
                     {
-                      "name": "bedr-1-o4",
+                      "name": "encounterDataValidationEntity-o4",
                       "type": "choice",
                       "label": "EQRO"
                     },
                     {
-                      "name": "bedr-1-o5",
+                      "name": "encounterDataValidationEntity-o5",
                       "type": "choice",
                       "label": "Other third-party vendor"
                     },
                     {
-                      "name": "bedr-1-o6",
+                      "name": "encounterDataValidationEntity-o6",
                       "type": "choice",
                       "label": "Proprietary system(s)",
                       "children": [
                         {
-                          "id": "bedr-2",
+                          "id": "encounterDataHippaCompliance",
```

Note the 2 `A`s in HIPAA as well.
```suggestion
                          "id": "encounterDataValidationSystemHipaaCompliance",
```